### PR TITLE
Only support Ruby 2.2+ and JRuby 9.0.0.0+

### PR DESCRIPTION
**Pull request by benlangfeld**
Drop attempts at support for Rubinius

### CODE REVIEW

**Comment by bklang** [comment]
Does JRuby 1.7 support Ruby 2.2 syntax?

**Comment by benlangfeld** [comment]
No, that is why I specified JRuby 9.0.0.0+

**Comment by emcgee** [comment]
FYI, I've seen a lot of segfault errors with Ruby 2.2.2 - https://github.com/celluloid/celluloid/issues/625#issuecomment-105080835

A simple reproduction was just to let ahn connect via Rayo to FS and then just idle. Eventually it would di with the error described above.

Haven't had time to dig further into it and report a bug back to ruby-lang or similar.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,13 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - jruby
-  - rbx-2.1.1
+  - 2.2.2
```
Comment (JustinAiken) [inline .travis.yml]: Why no 2.2.0 or 2.2.1 ?

**Comment by JustinAiken** [comment]
I like it! Rails 5 and Adhearsion 3, helping kill off old Rubies one travis matrix at a time!

File: .travis.yml (hunk 1 above)
Comment (benlangfeld) [inline .travis.yml]: Fixed.

**Comment by benlangfeld** [comment]
@emcgee I'm blocking this on https://github.com/adhearsion/adhearsion/pull/527. Hopefully that should help. I'd appreciate your feedback again once that is done.

**Comment by benlangfeld** [comment]
@emcgee The upgrade to Celluloid 0.16 is complete, and so this is now unblocked. I'd appreciate your feedback on how things look now.

**Comment by emcgee** [comment]
@benlangfeld I'll put this onto a staging server immediately that sees light usage and monitor it.

**Comment by bklang** [comment]
@emcgee Heads up that Adhearsion's develop branch is now Adhearsion 3 and contains breaking changes. If you want to test this, make sure you're using the `support/2.x.x` branch.

@benlangfeld Did the appropriate merge get to `support/2.x.x`? Is that even possible?

**Comment by benlangfeld** [comment]
No, `support/2.x.x` will not receive these changes, as this would require coordination with Punchblock versioning.

**Comment by emcgee** [comment]
@bklang Not a problem - saw those changes and I'll make the modifications on this side. Can't stand in the way of progress.

**Comment by emcgee** [comment]
@benlangfeld Follow up on this - I ran a stripped down version of our platform on this for the last 4 days on `2.2.2` with no random segfaults. Much other was broken, naturally, but calls completed as ussual with no issues. I'd consider https://github.com/celluloid/celluloid/issues/625#issuecomment-105080835 to be closed by this.

**Pull request merged**